ReText/__init__.py

```diff
@@ -71,6 +71,8 @@ def getBundledIcon(iconName):
 	'defaultMarkup': markups.MarkdownMarkup.name,
 	'defaultPreviewState': 'editor',
 	'detectEncoding': True,
+	'directoryPath': expanduser("~"),
+	'directoryTree': False,
```

Comment:
Maybe this option should be named `showDirectoryTree` to make it more obvious that it's a true/false option?

Also please document the new options in `configuration.md`.